css/styles.css

```diff
@@ -0,0 +1,152 @@
+:root {
+  --black: rgb(10, 10, 13);
+  --light-purple: rgb(192, 202, 245);
+  --light-purple-50: rgba(192, 202, 245, 0.5);
+  --teal: rgb(89, 253, 198);
+  --teal-30: rgba(89, 253, 198, 0.3);
+  --dark-blue: rgb(54, 59, 84);
+  --dark-blue-50: rgba(54, 59, 84, 0.5);
+  --light-blue: rgb(134, 216, 250);
+  --light-blue-50: rgba(134, 216, 250, 0.5);
+}
+
+body {
+  font-family: Satoshi-Regular, sans-serif;
+  background-color: black;
+  color: var(--light-purple);
+}
+
+h1,
+h2 {
+  font-family: ClashDisplay-Regular, sans-serif;
+  font-weight: 400;
+}
+
+a {
+  cursor: pointer;
+}
+
+#search {
+  width: 100%;
+}
+
+@media screen and (min-width: 768px) {
+  #search {
+    width: 60%;
+  }
+}
```

Comment:
I would move this to the bottom of the file. It's common practice to move media queries to the bottom of the file so they follow the cascade correctly